Payjoin-spec.md

```diff
@@ -222,6 +222,34 @@ If Alice pays Bob, she might be tempted to pay him a round amount, like `1.23000
 
 For this reason, during a [spare change](#spare-change) situation, we randomly round the amount in the output added by the receiver to the payjoin transaction.
 
+## Risks
+
+### On the receiver side: UTXO probing attack
+
+When the receiver creates a payjoin proposal, he exposes one or several inputs belonging to him.
+
+An attacker could create lot's of original transaction in order to learn the UTXOs of the receiver, while not broadcasting the payjoin proposal.
+
+We cannot fully prevent this type of attack, we implemented the following mitigations:
```

Comment:
` While we cannot prevent this type of attack entirely, we implemented the following mitigations:`